internal/databases/postgres/timeline.go

```diff
@@ -162,7 +163,8 @@ func ParseTimelineFromBackupName(backupName string) (uint32, error) {
 		return 0, newIncorrectBackupNameError(backupName)
```

Comment:
Add a couple of unit tests for new functionality regarding the possible backup name prefix.